app/pb_kits/playbook/pb_list/docs/_list_borderless.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= pb_rails("list", props: {ordered:false, dark:false, borderless:true}) do %>
```

Comment:
Is Rubocop enabled on this project? I'd think it would complain about missing space between `prop:value`.

Reply:
We should get it added I think. @drborges Who could help us get started? @garettarrowood ?

Reply:
I'm always happy to work with rubocop :). Last I talked with @markiearnold , I think the decision was to not include it here. If the UX/UI team wants to briefly talk it over, I'd be happy to explain the drawbacks and benefits of standardizing some style conventions in playbook. It can be frustrating if you don't know how to work with it. But the learning curve is pretty small.

Reply:
@garettarrowood I'll ping with you in Connect. We are planning on adding SASS lint & JS Lint to this project

Reply:
@kre8sions Can you add spaces between the props in the meantime?